guides/assets/stylesheets/main.css

```diff
@@ -254,24 +255,26 @@ body {
 }
 
 .clipboard-button:active {
-  background: #ccc;
+  background: var(--color-bg-clipboard-active);
 }
 
 #header {
-  background: #c52f24 url(../images/header_tile.gif) repeat-x;
+  background: #c52f24;
+  border-top: 2px ridge #bf2a20bb;
   color: #FFF;
   padding: 1.5em 0;
   z-index: 99;
 }
 
 #feature {
-  background: #d5e9f6 url(../images/feature_tile.gif) repeat-x;
```

Comment:
So... this change might be controversial or universally loved... unclear..

Basically, this was a 2px border that existed for the light theme that didn't for the dark theme, and to avoid making a second version for each theme, I replaced the image (again, it was a just `background` acting like a `border`) by using a `ridge` border with a bit of transparency to get a similar effect.  There is a minor difference in output, but I assume most won't even notice it at all.